### ISSUE TITLE
recipient field too short in private mail

### ISSUE THREAD

**Issue by christiansacks** [NuSkooler/enigma-bbs#243]
When I just tried to NetMail to an address, the recipient field seems to be limited to not enough characters as you have to add @1234:123/123 to the recipient field, and some contacts i have would have usernames with two words at least. 

So for example "Black Panther@1234:2/468" would be too long. how can I extend the field length?

**Comment by NuSkooler** [comment]
Until this is fixed for everyone, you can adjust your `theme.hjson`. Most of them are set like this in the default `luciano_blocktronics` theme:
```
TL1: { width: 19, textOverflow: "..." }
```

Change e.g. to:
```
TL1: { width: 32, textOverflow: "..." }
```

**Comment by NuSkooler** [comment]
Pushed a fix to `0.0.10-alpha`. The change is only in the `menu.hjson` **template**, so anyone having this issue will need to merge over the changes. They are minor, basically this was added to `ET2` of all menus using the `msg_area_post_fse` module:
```
maxLength: 36
```
